docs/setup-indexing.md

```diff
@@ -0,0 +1,55 @@
+# How to add indexing to your Helix 3 project step by step
+
+This document explains how to setup indexing in your Helix 3 project.
+
+## Setting up an initial query index
+
+If you start with the `helix-pages-starter` project you will have a `helix-query.yaml` configuration
+file in your GitHub repository. After having setup your `fstab.yaml` with a mountpoint that points
+into your SharePoint site or Google Drive, select a folder where your query index should be located,
+and enter the path relative to your mountpoint, e.g.:
+
+- `/query-index.xlsx` for a Sharepoint Site
+- `/query-index` for Google Drive
+
+For Sharepoint, the indexer will automatically setup the Excel workbook. For Google Drive, however,
+you have to create the spreadsheet yourself:
+
+1. Create the spreadsheet at the location specified above
+2. In that spreadsheet, create a sheet named `raw_index`
+3. In that sheet, enter the names of the properties in your `helix-query.yaml` in the first row -
+   one property name per cell - and add an entry `path`.
+
+## Setting up the properties that should be added to the index
+
+In order to decide what properties should be indexed in your HTML pages, look at the existing
+`properties` node in the starter `helix-query.yaml`. Further information on their semantics can be
+found in the [Reference](https://github.com/adobe/helix-index-files#reference) section of the
+`helix-index-files` project.
+
```

Comment:
also add a section explaining how to use `hlx up --print-index` to quickly configure your `helix-query.yaml`

Reply:
you can add a link to the mp4:
https://main--helix-website--adobe.hlx3.page/media_15501e6b07101255cb6ef99077f218666809ec354.mp4

( I already uploaded it to the helix-website mediabus)